ci-operator/step-registry/ipi/alibaba/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
After removing the symlink from ../OWNERS, this should probably use the OWNERS_ALIASES. I'm not sure why other places in this repository don't do this.

```
# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
# This file just uses aliases defined in OWNERS_ALIASES.

approvers:
  - alibaba-approvers
reviewers:
  - alibaba-approvers
```